test/snapshots/prefer-spread.js.md

```diff
@@ -531,7 +531,7 @@ Generated by [AVA](https://avajs.dev).
 > Output
 
     `␊
-      1 | [1, 2]␊
+      1 | [1, 2,]␊
```

Comment:
Shouldn't this be:
```suggestion
      1 | [1, , 2]␊
```

?

Reply:
No,

```text
> [1,].concat(2)
[ 1, 2 ]
> [1, , 2]
[ 1, <1 empty item>, 2 ]
```

It fixed in this way, because `2` inserted after `1`, and the old `[1,]` has trailing comma, we keep it, so `[1, 2,]` is correct.

Reply:
Ok. I misunderstood how concat works with holes. I assumed `[1,]` would create `1` + a hole, but you need `[1,,]` to get a hole.